osxkeychain/osxkeychain_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build darwin && cgo
+// +build darwin,cgo
```

Comment:
We can remove the old (`// +build ...`) format, as we no longer support go < 1.17